extensions/default/src/Components/CloudServerConfigurationComponent.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { ReactElement, useCallback, useEffect, useState } from 'react';
+import { useModal } from '@ohif/ui';
+
+function CloudServerConfigurationModalComponent({
+  servicesManager,
+  extensionManager,
+  onHide,
+}) {
+  const { customizationService } = servicesManager.services;
+
+  const [cloudServerAPI, setCloudServerAPI] = useState();
+
+  const [itemList, setItemList] = useState([]);
+
+  const [selectedItems, setSelectedItems] = useState([]);
+
+  const [itemLabels, setItemLabels] = useState([]);
+
+  useEffect(() => {
+    const activeDataSourceDef = extensionManager.getActiveDataSourceDef();
```

Comment:
Get the active data source. Determine if it is a cloud server API and if so create the cloud server API to configure it via a customization service implementation.